packages/store/src/index.ts

```
@@ -3,14 +3,16 @@ import { PrismaClient } from '@/generated/client'
-    },
+    // omit: {
+    //   user: { password_hash: true },
+    // },
   })
 
 declare global {
   var prisma:
-    | PrismaClient<{ omit: { user: { password_hash: true } } }>
+    | PrismaClient<{
+        // omit: { user: { : true } }
+      }>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)